src/main/java/com/twilio/oai/TwilioNodeGenerator.java

```diff
@@ -102,16 +101,19 @@ public OperationsMap postProcessOperationsWithModels(final OperationsMap objs, L
 
     private ApiResources generateResources(final List<CodegenOperation> opList) {
         final IConventionMapper conventionMapper = new LanguageConventionResolver(CONFIG_NODE_JSON_PATH);
-        final CodegenModelResolver codegenModelResolver = new CodegenModelResolver(conventionMapper,
+        final NodeCodegenModelResolver nodeCodegenModelResolver = new NodeCodegenModelResolver(conventionMapper,
                                                                                    modelFormatMap,
                                                                                    List.of(EnumConstants.NodeDataTypes.values()));
+        final CodegenModelResolver codegenModelResolver = new CodegenModelResolver(conventionMapper,
+                                                                                    modelFormatMap,
+                                                                                    List.of(EnumConstants.NodeDataTypes.values()));
 
         NodeApiResourceBuilder nodeApiResourceBuilder = new NodeApiResourceBuilder(actionTemplate, opList, allModels, directoryStructureService
                 , twilioCodegen.getToggles(JSON_INGRESS));
 
         nodeApiResourceBuilder.updateApiPath()
             .updateTemplate()
-            .updateOperations(new NodeParameterResolver(conventionMapper, codegenModelResolver))
+            .updateOperations(new NodeParameterResolver(conventionMapper, nodeCodegenModelResolver))
             .updateResponseModel(new LanguagePropertyResolver(conventionMapper), codegenModelResolver);
```

Comment:
why are we not using nodeCodegenModelResolver here in updateResponseModel

Reply:
So for request model, I have handled the resolving via Node specific resolvers, but for response model, it is handled in a different way, so it is getting overwritten wrongly. That's why I had to use common model resolver.